### ISSUE TITLE
Add sneep (Nina Baumgarten et al) to investigate if a SNP is probably destroying a TF motif

### ISSUE THREAD

**Issue by Firestar93** [biomedbigdata/TF-Prioritizer#65]

**Comment by nictru** [comment]
Requires #61 to be finished first

**Comment by nictru** [comment]
I assume the workflow will look approximately like this:

1. Identify motif regions based on STARE input regions using FIMO (#61)
2. Fetch VCF file containing all known SNPs for a given genome
3. Intersect motif regions with SNPs
4. Use remaining SNPs as input for Sneep
5. Include the Sneep output to the report

**Comment by Firestar93** [comment]
Yes, just that the regions that we search with FIMO are only predicted and prioritized enhancers and promoters from STARE not all regions. Also: in these enhancer and promoter regions we only search for motifs with FIMO for TFs that were prioritized.
